### DISCUSSION TITLE
timezone environment variable

### DISCUSSION THREAD

**Discussion by strider-84** [hertzg/rtl_433_docker#84] [Q&A] [answered]
hello,
my local timezone and utc timezone are different. When i pass the time metadata with tz option (-M "time:tz"), i still get utc timestamp in my output? How do i pass my local timezone ? or am i using time metadata wrongly?
please advice,
Thanks!

**Comment by hertzg** [comment]
This question would be targeted towards the rtl_433 tool itself, better ask there.

https://github.com/merbanan/rtl_433

**Reply by strider-84** [reply, marked as answer]
Thanks for your reply!
If i install rtl_433 without a docker (by merbanan) on my machine, i get the correct timestamp using the :tz metadata option. But i dont if i run it as a docker container (this one), hence asking the question here.

Anyways, i was able to solve it eventually by passing -e TZ=America/Los_Angeles argument in a docker run command.

Thanks!
